WordPress/WordPressTest/BlogDetailsViewController+MeButtonTests.swift

```diff
@@ -66,13 +66,21 @@ class BlogDetailsViewControllerTests: XCTestCase {
             return
         }
         controller.addMeButtonToNavigationBar()
-        guard let meButton = controller.navigationItem.rightBarButtonItem else {
+        guard controller.navigationItem.rightBarButtonItem != nil else {
             XCTFail("Me Button not installed")
             return
         }
         scenePresenter?.presentExpectation = expectation(description: "Me was presented")
         // When
-        _ = meButton.target?.perform(meButton.action)
+
+        guard let target = blogDetailsViewController?.target(forAction: #selector(BlogDetailsViewController.presentHandler), withSender: blogDetailsViewController) else {
+            XCTFail("Cippa")
```

Comment:
I'm thinking this was meant to be replaced with a meaningful fail message. 😄

Reply:
oooooppsssss yes it was :)